docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -72,6 +87,8 @@ export default function PlainCssPriority() {
 }
 ```
 
+**Note:** If you are using styled-components and have `StyleSheetManager` with custom `target`, make sure that the `target` will be the first element in the `head` of your app. If you are curious to see how it can be done, you can take a look on the `StylesProvider` implementation in the `@material-ui/styled-engine-sc` package.
```

Comment:
```suggestion
**Note:** If you are using styled-components and have `StyleSheetManager` with a custom `target`, make sure that the target is the first element in the HTML `<head>`. If you are curious to see how it can be done, you can take a look on the `StylesProvider` implementation in the `@material-ui/styled-engine-sc` package.
```